game/scripts/vscripts/components/progression/hero_progression.lua

```diff
@@ -1,48 +1,73 @@
 if HeroProgression == nil then
     HeroProgression = class({})
     Debug.EnabledModules['progression:*'] = true
+
```

Comment:
atom.io -- I've setup the editorconfig already, so it's gonna be 2 spaces from now on. I am using 4 spaces everywhere else, that's why.